test/regression/schedule

```diff
@@ -17,4 +17,4 @@ test: query_filter
 test: temporary_tables
 test: altered_tables
 test: transaction_errors
-test: secrets
+test: secrets
```

Comment:
Can you install https://editorconfig.org/ and save this file and your other files again. Then this diff should disappear, because a newline will be included at the end of the file.